validation/jinja2/validation/segment_details.html

```diff
@@ -48,7 +48,7 @@
         <button class="button button--success button--back" onclick='goBack()'>Back</button>
         {% if auth %}
             <button data-cy="edit-button" class="button button--fail button--right button--edit"
-                onclick="showDiv('{{ segment_name }}')"
+                onclick='showDiv("{{ segment_name }}")'
```

Comment:
```suggestion
                onclick='showSegmentDiv()'
                data-segment-name="{{ segment_name }}"
```

With the appropriate function defined _something_ like this:

```javascript
function showSegmentDiv() {
    const segmentName = this.target.dataset.segmentName;
    return showDiv(segmentName);
}
```

Reply:
I think this will work